src/mumble/MainWindow.ui

```diff
@@ -887,6 +890,39 @@ the channel's context menu.</string>
     <bool>false</bool>
    </property>
   </action>
+  <action name="qaChannelFilterVisibilityAlways">
+   <property name="checkable">
+    <bool>true</bool>
+   </property>
+   <property name="text">
+    <string>&amp;Always show channel</string>
+   </property>
+   <property name="toolTip">
+    <string>Always show channel when filtering. Even if it is empty.</string>
+   </property>
+  </action>
+  <action name="action_Hide_channel">
+   <property name="checkable">
+    <bool>true</bool>
+   </property>
+   <property name="text">
+    <string>&amp;Hide channel if empty</string>
+   </property>
+   <property name="toolTip">
+    <string>When filtering channel gets hidden unless it contains users.</string>
+   </property>
+  </action>
+  <action name="qaChannelFilterVisibilityNormal">
+   <property name="checkable">
+    <bool>true</bool>
+   </property>
+   <property name="text">
+    <string>&amp;Default</string>
+   </property>
+   <property name="toolTip">
+    <string>Apply channel default behavior.</string>
```

Comment:
Maybe:

> Don't apply any special channel filtering rules to this channel. 

?

I dunno. "Apply default channel behavior" sounds confusing to me. I would expect it to explain what that default behavior is.